be/src/vec/functions/simple_function_factory.h

```diff
@@ -86,6 +86,7 @@ void register_function_hex_variadic(SimpleFunctionFactory& factory);
 
 void register_function_url(SimpleFunctionFactory& factory);
 
+void register_function_has_all(SimpleFunctionFactory& factory);
```

Comment:
should be registered in `be/src/vec/functions/array/function_array_register.cpp`